data-table-row.html

```diff
@@ -51,7 +51,8 @@
         expandDetails: {
           type: Boolean,
           notify: true,
-          value: false
+          value: false,
+          reflectToAttribute: true
```

Comment:
need this property to reflect to attribute so that css can be used for a row expand indicator